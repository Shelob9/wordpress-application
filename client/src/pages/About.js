import * as React from 'react'
import { NavLink } from 'react-navi'

export default function About() {
	return (
		<div>
			<h2>This is the about page</h2>
			<p>This route was compiled and handled by Navi, including all the heavy lifting for SEO, creating sitemaps including this page, code-splitting, etc!</p>
		</div>
	)
}
