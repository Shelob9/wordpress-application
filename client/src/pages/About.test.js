import React from "react";
import {create} from "react-test-renderer";
import About from "./About";

describe( 'About component', () => {
	it( 'matches snapshot', () => {
		expect( create(<About/>).toJSON() ).toMatchSnapshot()
	});
});
