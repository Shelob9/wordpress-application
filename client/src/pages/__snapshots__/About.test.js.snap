// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot 1`] = `
<div>
  <h2>
    This is the about page
  </h2>
  <p>
    This route was compiled and handled by Navi, including all the heavy lifting for SEO, creating sitemaps including this page, code-splitting, etc!
  </p>
</div>
`;
